Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,149 @@
+variables:
+  SAM_TEMPLATE: {{cookiecutter.sam_template}}
+  AWS_ACCESS_KEY_ID_: ${AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY_: ${AWS_SECRET_ACCESS_KEY}
+  TESTING_STACK_NAME: {{cookiecutter.testing_stack_name}}
+  TESTING_REGION: {{cookiecutter.testing_region}}
+  TESTING_PIPELINE_EXECUTION_ROLE: {{cookiecutter.testing_pipeline_execution_role}}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.testing_cloudformation_execution_role}}
+  TESTING_ARTIFACTS_BUCKET: {{cookiecutter.testing_artifacts_bucket}}
+  TESTING_ECR_REPO: {{cookiecutter.testing_ecr_repo}}
+  PROD_STACK_NAME: {{cookiecutter.prod_stack_name}}
+  PROD_REGION: {{cookiecutter.prod_region}}
+  PROD_PIPELINE_EXECUTION_ROLE: {{cookiecutter.prod_pipeline_execution_role}}
+  PROD_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.prod_cloudformation_execution_role}}
+  PROD_ARTIFACTS_BUCKET: {{cookiecutter.prod_artifacts_bucket}}
+  PROD_ECR_REPO: {{cookiecutter.prod_ecr_repo}}
```

Comment:
same as testing-ECR, let's follow this https://github.com/aws/aws-sam-cli-pipeline-init-templates/blob/main/Jenkins/two-stage-pipeline-template/%7B%7Bcookiecutter.outputDir%7D%7D/Jenkinsfile#L24-L32

Reply:
Sure! Will update it